---
title: "Research"
date: 2025-01-04
meta: true
description: ""
---

_Last updated Sat, Jan 4 2024_

This page is currently being updated to more effectively represent everything that I've worked on over the years. Please bear with me while I finish it up!

## Protein engineering at [U. Chicago Molecular Engineering](https://pme.uchicago.edu/)
As a postdoctoral scientist in the [Mendoza Lab](https://pme.uchicago.edu/faculty/juan-mendoza), I use display-based libraries to engineer cytokines and their receptors to manipulate their signaling properties with the ultimate goal of designing next generation protein therapeutics. 

## Structural immunology at [U. Chicago Biology](https://biologicalsciences.uchicago.edu/)
As a graduate student in the [Adams Lab](https://ejadamslab.bsd.uchicago.edu/) I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the [archaic HLA-B*73:01](https://pubmed.ncbi.nlm.nih.gov/21868630/) and the non-classical [HLA-F](https://pubmed.ncbi.nlm.nih.gov/28636952/). Both molecules serve as ligands for the germline-encoded, polymorphic [Killer-immunoglobulin-like receptors (KIRs)](https://www.annualreviews.org/doi/abs/10.1146/annurev-immunol-042617-053149), but how this occurs and the consequences thereof were unclear when I began my work.

In the case of HLA-B\*73:01, it was known that similar to other class I molecules, HLA-B\*73:01 presents "short" peptides on the surface of cells. However, what I was interested was in what ways HLA-B\*73:01 was different to other class I molecules. To learn more about HLA-B\*73:01, check out our preprint [here](https://www.biorxiv.org/content/10.1101/2024.11.25.625330v1), while the manuscript is under review.

In the case of HLA-F, the questions were more fundamental. It was thought that unlike other class I molecules, HLA-F did not need to form a heterodimer with β2M in order to be expressed on the surface of cells. Charlie Dulberger, a former mentor of mine in the Adams lab, showed that HLA-F could also present peptides similar to all other class I molecules. So then which is the "true" form and what regulates the expression of these different forms? A preprint of my work on HLA-F is in preparation as well.

For the most thorough overview of my graduate work, feel free to check out [my thesis](https://knowledge.uchicago.edu/record/5721?v=pdf).

## *Plasmodium* computational genomics in the [Llinás Lab](https://www.llinaslab.psu.edu/)
As a member of the Llinás Lab I contributed to work in understanding transcriptional regulation in the deadliest human-infecting parasite that causes malaria, *Plasmodium falciparum*.

Primary contributions resulted in the following publications:
- [Red Blood Cell Invasion by the Malaria Parasite Is Coordinated by the PfAP2-I Transcription Factor](http://www.cell.com/cell-host-microbe/fulltext/S1931-3128(17)30199-3)
- [Refining the transcriptome of the human malaria parasite Plasmodium falciparum using amplification-free RNA-seq](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-06787-5)

Want to investigate our data yourself? Use this [Shiny App](https://pzross.shinyapps.io/shinyplasmo/) I developed for doing so!  
You should also check out the generous commentary on our study here: [Scientists on a RAMPAGE to find apicomplexan transcription start sites](https://www.nature.com/articles/s41579-021-00587-8)

Additional contributions resulted in the following publications:
- [PfSR1 controls alternative splicing and steady-state RNA levels in Plasmodium falciparum through preferential recognition of specific RNA motifs](http://onlinelibrary.wiley.com/doi/10.1111/mmi.13007/abstract)
- [A single nucleotide polymorphism in a Plasmodium berghei ApiAP2 transcription factor alters the development of host immunity](https://advances.sciencemag.org/content/6/6/eaaw6957.abstract)
