<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Research | Phil Ross, Ph.D.</title>
<meta name="keywords" content="">
<meta name="description" content="Last updated Sat, Jan 4 2024
Cytokine engineering in the Mendoza Lab
Coming soon!
Molecular immunology in the Adams Lab
During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the archaic HLA-B*73:01 and the non-classical HLA-F. Both molecules serve as ligands for the germline-encoded, polymorphic Killer-immunoglobulin-like receptors (KIRs), but it was unclear exactly how.
In the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level.">
<meta name="author" content="Phil Ross, Ph.D.">
<link rel="canonical" href="https://www.philippzross.com/research/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.c5cae7d01173918ab947eeb69fa188e30d06b509c152e9062dfbc23445272c60.css" integrity="sha256-xcrn0BFzkYq5R&#43;62n6GI4w0GtQnBUukGLfvCNEUnLGA=" rel="preload stylesheet" as="style">
<link rel="icon" href="https://www.philippzross.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://www.philippzross.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://www.philippzross.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://www.philippzross.com/apple-touch-icon.png">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<link rel="alternate" hreflang="en" href="https://www.philippzross.com/research/">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
</noscript><meta property="og:title" content="Research" />
<meta property="og:description" content="Last updated Sat, Jan 4 2024
Cytokine engineering in the Mendoza Lab
Coming soon!
Molecular immunology in the Adams Lab
During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the archaic HLA-B*73:01 and the non-classical HLA-F. Both molecules serve as ligands for the germline-encoded, polymorphic Killer-immunoglobulin-like receptors (KIRs), but it was unclear exactly how.
In the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://www.philippzross.com/research/" /><meta property="article:section" content="" />
<meta property="article:published_time" content="2025-01-04T00:00:00+00:00" />
<meta property="article:modified_time" content="2025-01-04T00:00:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Research"/>
<meta name="twitter:description" content="Last updated Sat, Jan 4 2024
Cytokine engineering in the Mendoza Lab
Coming soon!
Molecular immunology in the Adams Lab
During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the archaic HLA-B*73:01 and the non-classical HLA-F. Both molecules serve as ligands for the germline-encoded, polymorphic Killer-immunoglobulin-like receptors (KIRs), but it was unclear exactly how.
In the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "Research",
      "item": "https://www.philippzross.com/research/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Research",
  "name": "Research",
  "description": "Last updated Sat, Jan 4 2024\nCytokine engineering in the Mendoza Lab Coming soon!\nMolecular immunology in the Adams Lab During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the archaic HLA-B*73:01 and the non-classical HLA-F. Both molecules serve as ligands for the germline-encoded, polymorphic Killer-immunoglobulin-like receptors (KIRs), but it was unclear exactly how.\nIn the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level.\n",
  "keywords": [
    
  ],
  "articleBody": "Last updated Sat, Jan 4 2024\nCytokine engineering in the Mendoza Lab Coming soon!\nMolecular immunology in the Adams Lab During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the archaic HLA-B*73:01 and the non-classical HLA-F. Both molecules serve as ligands for the germline-encoded, polymorphic Killer-immunoglobulin-like receptors (KIRs), but it was unclear exactly how.\nIn the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level.\nA preprint for this project is currently available here.\nIn the case of HLA-F, it was not clear in which conformation HLA-F actually carries out its functions. Previous studies had shown that unlike other class I molecules, HLA-F did not need to form a heterodimer with β2M in order to be expressed at the cell-surface. But more recently, the Adams Lab showed that HLA-F could also present peptides similar to all other class I molecules in humans. Thus the questions for this project were more fundamental: which forms of HLA-F are functional and what regulates the expression of these different forms?\nA preprint on HLA-F is in preparation as well.\nFor the most thorough overview of my graduate work, feel free to check out my thesis.\nPlasmodium computational genomics in the Llinás Lab As a member of the Llinás Lab I contributed to work in understanding transcriptional regulation in the deadliest human-infecting parasite that causes malaria, Plasmodium falciparum.\nPrimary contributions resulted in the following publications:\nRed Blood Cell Invasion by the Malaria Parasite Is Coordinated by the PfAP2-I Transcription Factor Refining the transcriptome of the human malaria parasite Plasmodium falciparum using amplification-free RNA-seq Want to investigate our data yourself? Use this Shiny App I developed for doing so!\nYou should also check out the generous commentary on our study here: Scientists on a RAMPAGE to find apicomplexan transcription start sites\nAdditional contributions resulted in the following publications:\nPfSR1 controls alternative splicing and steady-state RNA levels in Plasmodium falciparum through preferential recognition of specific RNA motifs A single nucleotide polymorphism in a Plasmodium berghei ApiAP2 transcription factor alters the development of host immunity ",
  "wordCount" : "395",
  "inLanguage": "en",
  "datePublished": "2025-01-04T00:00:00Z",
  "dateModified": "2025-01-04T00:00:00Z",
  "author":{
    "@type": "Person",
    "name": "Phil Ross, Ph.D."
  },
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://www.philippzross.com/research/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Phil Ross, Ph.D.",
    "logo": {
      "@type": "ImageObject",
      "url": "https://www.philippzross.com/favicon.ico"
    }
  }
}
</script>



<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/katex.min.css" integrity="sha384-wcIxkf4k558AjM3Yz3BBFQUbk/zgIYC2R0QpeeYb+TwlBVMrlgLqwRjRtGZiK7ww" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/katex.min.js" integrity="sha384-hIoBPJpTUs74ddyc4bFZSM1TVlQDA60VBbJS0oA934VSz82sBx1X7kSx2ATBDIyd" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/contrib/auto-render.min.js" integrity="sha384-43gviWU0YVjaDtb/GhzOouOXtZMP/7XUzwPTstBeZFe/+rCMvRwr4yROQP43s0Xk" crossorigin="anonymous"
  onload="renderMathInElement(document.body);"></script>

<script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement(document.body, {
          delimiters: [
            {left: '$$', right: '$$', display: true},
            {left: '$', right: '$', display: false},
            {left: "\\begin{equation}", right: "\\end{equation}", display: true},
            {left: "\\begin{equation*}", right: "\\end{equation*}", display: true},
            {left: "\\begin{align}", right: "\\end{align}", display: true},
            {left: "\\begin{align*}", right: "\\end{align*}", display: true},
            {left: "\\begin{alignat}", right: "\\end{alignat}", display: true},
            {left: "\\begin{gather}", right: "\\end{gather}", display: true},
            {left: "\\begin{CD}", right: "\\end{CD}", display: true},
          ],
          throwOnError : false
        });
    });
</script>
 


</head>

<body class="" id="top">

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://www.philippzross.com/" accesskey="h" title="Phil Ross">
                <img src="https://www.philippzross.com/favicon.ico" alt="" aria-label="logo"
                    height="18"
                    width="18">Phil Ross</a>
            <div class="logo-switches">
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="https://www.philippzross.com/research/" title="Research">
                    <span class="active">Research</span>
                </a>
            </li>
        </ul>
    </nav>
</header>

    <main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title entry-hint-parent">
      Research
    </h1>
  </header> 
  <div class="post-content"><p><em>Last updated Sat, Jan 4 2024</em></p>
<h2 id="cytokine-engineering-in-the-mendoza-labhttpsvoicesuchicagoeduthemendozagroup">Cytokine engineering in the <a href="https://voices.uchicago.edu/themendozagroup/" target="_blank">Mendoza Lab</a></h2>
<p>Coming soon!</p>
<h2 id="molecular-immunology-in-the-adams-labhttpejadamslabbsduchicagoedu">Molecular immunology in the <a href="http://ejadamslab.bsd.uchicago.edu/" target="_blank">Adams Lab</a></h2>
<p>During my Ph.D. in the Adams Lab, I worked mainly on two proteins that play important roles in the innate and adaptive immune response: the <a href="https://pubmed.ncbi.nlm.nih.gov/21868630/" target="_blank">archaic HLA-B*73:01</a> and the non-classical <a href="https://pubmed.ncbi.nlm.nih.gov/28636952/" target="_blank">HLA-F</a>. Both molecules serve as ligands for the germline-encoded, polymorphic <a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-immunol-042617-053149" target="_blank">Killer-immunoglobulin-like receptors (KIRs)</a>, but it was unclear exactly how.</p>
<p>In the case of HLA-B*73:01, it was clear that similar to other class I molecules, HLA-B*73:01 presents short peptides to T and NK cell receptors that may either allow or prevent receptor recognition. The real question was whether HLA-B*73:01 was unique in any way relative to other class I alleles at the molecular level.</p>
<p>A preprint for this project is currently available <a href="https://www.biorxiv.org/content/10.1101/2024.11.25.625330v1" target="_blank">here</a>.</p>
<p>In the case of HLA-F, it was not clear in which conformation HLA-F actually carries out its functions. Previous studies had shown that unlike other class I molecules, HLA-F did not need to form a heterodimer with β2M in order to be expressed at the cell-surface. But more recently, the Adams Lab showed that HLA-F could also present peptides similar to all other class I molecules in humans. Thus the questions for this project were more fundamental: which forms of HLA-F are functional and what regulates the expression of these different forms?</p>
<p>A preprint on HLA-F is in preparation as well.</p>
<p>For the most thorough overview of my graduate work, feel free to check out <a href="https://knowledge.uchicago.edu/record/5721?v=pdf" target="_blank">my thesis</a>.</p>
<h2 id="plasmodium-computational-genomics-in-the-llinás-labhttpswwwllinaslabpsuedu"><em>Plasmodium</em> computational genomics in the <a href="https://www.llinaslab.psu.edu/" target="_blank">Llinás Lab</a></h2>
<p>As a member of the Llinás Lab I contributed to work in understanding transcriptional regulation in the deadliest human-infecting parasite that causes malaria, <em>Plasmodium falciparum</em>.</p>
<p>Primary contributions resulted in the following publications:</p>
<ul>
<li><a href="http://www.cell.com/cell-host-microbe/fulltext/S1931-3128%2817%2930199-3" target="_blank">Red Blood Cell Invasion by the Malaria Parasite Is Coordinated by the PfAP2-I Transcription Factor</a></li>
<li><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-06787-5" target="_blank">Refining the transcriptome of the human malaria parasite Plasmodium falciparum using amplification-free RNA-seq</a></li>
</ul>
<p>Want to investigate our data yourself? Use this <a href="https://pzross.shinyapps.io/shinyplasmo/" target="_blank">Shiny App</a> I developed for doing so!<br>
You should also check out the generous commentary on our study here: <a href="https://www.nature.com/articles/s41579-021-00587-8" target="_blank">Scientists on a RAMPAGE to find apicomplexan transcription start sites</a></p>
<p>Additional contributions resulted in the following publications:</p>
<ul>
<li><a href="http://onlinelibrary.wiley.com/doi/10.1111/mmi.13007/abstract" target="_blank">PfSR1 controls alternative splicing and steady-state RNA levels in Plasmodium falciparum through preferential recognition of specific RNA motifs</a></li>
<li><a href="https://advances.sciencemag.org/content/6/6/eaaw6957.abstract" target="_blank">A single nucleotide polymorphism in a Plasmodium berghei ApiAP2 transcription factor alters the development of host immunity</a></li>
</ul>

  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2025 <a href="https://www.philippzross.com/">Phil Ross, Ph.D.</a></span> ·     
    <span>
    Powered by 
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/pmichaillat/hugo-website/" rel="noopener" target="_blank">a modified version</a>
         of 
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    document.querySelectorAll('pre > code').forEach((codeblock) => {
        const container = codeblock.parentNode.parentNode;

        const copybutton = document.createElement('button');
        copybutton.classList.add('copy-code');
        copybutton.innerHTML = 'copy';

        function copyingDone() {
            copybutton.innerHTML = 'copied!';
            setTimeout(() => {
                copybutton.innerHTML = 'copy';
            }, 2000);
        }

        copybutton.addEventListener('click', (cb) => {
            if ('clipboard' in navigator) {
                navigator.clipboard.writeText(codeblock.textContent);
                copyingDone();
                return;
            }

            const range = document.createRange();
            range.selectNodeContents(codeblock);
            const selection = window.getSelection();
            selection.removeAllRanges();
            selection.addRange(range);
            try {
                document.execCommand('copy');
                copyingDone();
            } catch (e) { };
            selection.removeRange(range);
        });

        if (container.classList.contains("highlight")) {
            container.appendChild(copybutton);
        } else if (container.parentNode.firstChild == container) {
            
        } else if (codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.nodeName == "TABLE") {
            
            codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.appendChild(copybutton);
        } else {
            
            codeblock.parentNode.appendChild(copybutton);
        }
    });
</script>
</body>
</html>
